lib/src/code/code.dart

```diff
@@ -1,20 +1,32 @@
 import 'package:collection/collection.dart';
 import 'package:flutter/material.dart';
+import 'package:highlight/highlight.dart';
+import 'package:highlight/languages/dart.dart';
+import 'package:highlight/languages/go.dart';
+import 'package:highlight/languages/java.dart';
+import 'package:highlight/languages/python.dart';
+import 'package:highlight/languages/scala.dart';
 
 import 'code_line.dart';
 import 'text_range.dart';
 import 'tokens.dart';
 
+final List<Mode> slashCommentSignLanguages = [java, dart, go, scala];
```

Comment:
The cleanest way I can think of is
```dart
const _slashes = '//';
const _hash = '#';

const _singleLineComments = <String, List<String>>{
  {java: [_slashes]},
  {python: [_hash]},
  // ...
};
```

Then just pick from the map without `if`s.